packages/utils/upgrade/src/__tests__/core/format.test.ts

```diff
@@ -0,0 +1,85 @@
+import * as f from '../../core/format';
+
+import type { RunReports } from '../../types';
+
+describe('Format', () => {
+  test('Path', () => {
+    const formatted = f.path('Foo');
+    expect(formatted).toStrictEqual(formatted);
+  });
+
+  test.each(['4.15.0', '5.0.0', '11.11.11', 'latest', 'major', 'minor', 'patch'])(
+    'Version (%s)',
+    (version) => {
+      const formatted = f.version(version);
+      expect(formatted).toStrictEqual(formatted);
+    }
+  );
+
+  test.each(['>4.0.0 <=5.0.0', '>4.5.2 <7.1.4'])('Version Range (%s)', (range) => {
+    const formatted = f.versionRange(range);
+    expect(formatted).toStrictEqual(formatted);
+  });
+
+  test.each(['console.log-to-console.info', 'update-json-file', 'transform'])(
+    'Transform File Path (%s)',
+    (transformFilePath) => {
+      const formatted = f.transform(transformFilePath);
+      expect(formatted).toStrictEqual(formatted);
+    }
+  );
+
+  test.each(['foo', 'bar', 'baz'])('Highlight (%s)', (text) => {
+    const formatted = f.version(text);
+    expect(formatted).toStrictEqual(formatted);
+  });
+
+  test('Reports', () => {
+    const reports: RunReports = [
+      {
+        transform: {
+          path: './5.0.0/transform.code.ts',
+          fullPath: '/root/5.0.0/transform.code.ts',
+          version: '5.0.0',
+          formatted: 'transform',
+          kind: 'code',
+        },
+        report: { error: 0, skip: 1, nochange: 2, ok: 3, timeElapsed: '0.400' },
+      },
+      {
+        transform: {
+          path: './6.3.0/update-deps.json.ts',
+          fullPath: '/root/6.3.0/update-deps.json.ts',
+          version: '6.3.0',
+          formatted: 'update deps',
+          kind: 'json',
+        },
+        report: { error: 5, skip: 0, nochange: 90, ok: 40, timeElapsed: '0.030' },
+      },
+    ];
+
+    const formatted = f.reports(reports);
+
+    // Note: Check the jest terminal output (human-readable) in case the snapshot has changed
+    expect(formatted).toMatchInlineSnapshot(`
```

Comment:
Maybe add a note on the quickest way to update this snapshot? Or rewrite so instead of snapshot we just look for keywords in the right places? I expect it will change a lot during development

Reply:
Done :heavy_check_mark: